### PR TITLE
✅ TestsSlike Video Player - AMP Component

### PR DESCRIPTION
**Pull request by SlikeTech**
Slike Studio is a TIL's in-house audio-video platform to meet the enterprise needs, which provides live broadcast, on-demand video & podcast solutions. We created this leading Video Platform-as-a-Service, enabling different businesses to create any video solution and workflow, whether standalone or tightly coupled.

This AMP Component is a new component meant for embedding Slike media player, there are certain mandatory parameters as well as customised ones which we can use as per the user configurations required, details of which can be found inside.

### CODE REVIEW

**Comment by SlikeTech** [comment]
@googlebot I consent

**Comment by SlikeTech** [comment]
@jridgewell
I followed all the processes for ownership and CLA but not able out of it.
Please help.

**Comment by jridgewell** [comment]
Hi @SlikeTech, don't worry too much about the CLA. You've included multiple commits, with some commits being authored by yourself and some by @ranjan000. As long as both of you comment here that you consent, we can manually verify the CLA has been signed using the associated email addresses in the commit.

For now, I'm sending this to @ampproject/wg-ui-and-a11y for review.

File: extensions/amp-slike/validator-amp-slike.protoascii [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#
+
+tags: {  # amp-slike
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-slike"
+    version: "0.1"
+    version: "latest"
+    requires_usage: EXEMPTED
```
Comment (Gregable) [inline extensions/amp-slike/validator-amp-slike.protoascii:24]: Remove this line.

File: extensions/amp-slike/validator-amp-slike.protoascii [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    deprecated_allow_duplicates: true
```
Comment (Gregable) [inline extensions/amp-slike/validator-amp-slike.protoascii:25]: And this line.

File: extensions/amp-slike/validator-amp-slike.protoascii [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-slike>
+  html_format: AMP
+  tag_name: "AMP-SLIKE"
+  requires_extension: "amp-slike"
+   attrs: {
```
Comment (Gregable) [inline extensions/amp-slike/validator-amp-slike.protoascii:33]: nit. looks like whitespace alignment is off by one here, but I may be reading the diff incorrectly.

File: extensions/amp-slike/0.1/test/validator-amp-slike.out [hunk 4]
```diff
@@ -0,0 +1,21 @@
+PASS
```
Comment (Gregable) [inline extensions/amp-slike/0.1/test/validator-amp-slike.out:1]: Out file doesn't match the input file. You can regenerate the out file by running from the `validator/` dir

```
python build.py --update_tests
```

File: extensions/amp-slike/0.1/test/validator-amp-slike.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
```
Comment (Gregable) [inline extensions/amp-slike/0.1/test/validator-amp-slike.html:1]: Needs a copyright comment header and documentation. See similar `.html` test files in other extension directories.

**Review by Gregable — CHANGES_REQUESTED** [review]
My comments are for the validator changes only.

**Comment by ranjan000** [comment]
@google I consent.

**Comment by ranjan000** [comment]
@google I consent

**Comment by kristoferbaxter** [comment]
After a few mishaps with the iOS GitHub client, I've added Kevin to review.

**Comment by SlikeTech** [comment]
@google I consent

File: extensions/amp-slike/0.1/test/validator-amp-slike.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
+<!--
+  Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (krdwan) [inline extensions/amp-slike/0.1/test/validator-amp-slike.html:2]: nit: Copyright 2020

**Review by krdwan — APPROVED** [review]
LGTM

**Comment by SlikeTech** [comment]
@Gregable All the owners are on board. May I know the owner who is missing?

**Pull request closed without merging**